templates/etherpad/0/docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+etherpad:
+  image: moss/etherpad
+  environment:
+    EP_TITLE: ${etherpad_title}
+    FAVICON_URL: ${etherpad_favicon}
+    ADMIN_PASS: ${etherpad_admin_password}
+    DB_HOST: etherpad-db
+    DB_TYPE: mysql
+    DB_PORT: 3306
+    DB_USER: etherpad-db-user
+    DB_PASS: ${etherpad_db_password}
+    DB_NAME: etherpad-db-name
+  links:
+  - etherpad-db:mysql
```

Comment:
# Set the link to database service
--  - etherpad-db:mysql
++  - ${etherpad_db_service}:mysql